enhancements/windows-containers/container-runtime-containerd.md

```diff
@@ -0,0 +1,213 @@
+is part of kubelet code, runs as part of kubelet and is tightly coupled with kubelet's lifecycle. From kubernetes
+1.24 onwards dockershim will be removed from kubelet code. At present WMCO uses Docker CRI as the default runtime.
+Aim is to make containerd as default runtime and move away from Docker CRI before dokershim has been decoupled from
+kubelet.
+
+### Goals
+
+As part of this enhancement we plan to do the following:
+* Make containerd as default runtime.
+* When upgrade happens from WMCO 5.0 to WMCO 6.0, containerd will become default runtime.  
+
+### Non-Goals
+
+* De-configuring docker runtime is not part of this enhancement.
+* Refactoring or re-design of WMCO due to dockershim deprecation.
+* Allowing the user to choose container runtime with user facing option is not supported.
+*  Downgrade is not supported.
+
+## Proposal
+
+To make containerd as default runtime, containerd should be installed before kueblet and
+kubelet parameters need to be updated so that containerd will become default runtime. In
+upgrade case two scenarios need to be handled.
+1) MachineSet
+  Each Windows VM configured by previous versions of the WMCO has its Machine
+object deleted, resulting in the drain and deletion of the Windows node, and the termination
+then followed by a recreation of the VM. The upgraded WMCO instance will then be able to configure
+VMs that will be created with containerd as default runtime. All minimum requirements stated
+as part of WMCO upgrade will be applicable here as well. The golden image that we used to create
+Windows VM doesn't need to have docker runtime. It is optional as customers may use docker for
+non-kubernetes requirements.
+2) BYOH ( Bring Your Own Host)
+  If Windows VM was created by BYOH option, the upgrade process will uninstall all components
+that were installed by WMCO. As part of the de-configuring process, Windows OS specific configurations
+like HNS-Network that were created by WMCO would have been deleted. During reinstall workflow, containerd
+will be installed along with kubelet, kube-proxy, CNI and hybrid-overlay.
+
+There is no difference between MachineSet and BYOH on how we enable containerd as default runtime. 
+
+Containerd Migration plan
+   *Containerd will become default runtime as part of OpenShift 4.11
+   *For current usage, we plan to introduce WMCO feature flag and release it only for community operator.
+   *The default value will be Docker CRI. In 4.10 community branch, the default value will be containerd.
+   *Customers will not have the option to choose runtime by using WMCO feature flag.
+   *In BYOH case Windows VM Will be de-configured and re-configured with the WMCO supported runtime.
+   *In MachineSet option Docker CRI based VM will be deleted and a new VM will be created with containerd
+    as default runtime.
+
+### User Stories
+
+Stories can be found within the [Windows Containers: containerd epic](https://issues.redhat.com/browse/WINC-505)
+
+### Justification
```

Comment:
This section is also not rendering correctly. I also suggest adding two sub-sections, one justifying not picking Mirantis and the other justifying picking containerd.